slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -38,12 +38,26 @@ using viam::service::slam::v1::SLAMService;
 namespace viam {
 
 static const int checkForShutdownIntervalMicroseconds = 1e5;
+
+// rgb value representing no input from a Cario pained map
+static const unsigned char defaultCairosEmptyPaintedSlice = 102;
+static const int jpegQuality = 50;
+// byte limit on GRPC, used to help determine sampling skip_count
+static const float maximumGRPCByteLimit = 32 * 1024 * 1024;
+// coeffient to adjust the skip count for the PCD to ensure the file is within
+// grpc limitations
+static const float samplingFactor = 5;
```

Comment:
Was this 0.5 before? Was the change intentional?

Reply:
yeah, turns out I left a 0 off the byte limit variable, so I changed the sampling factor accordingly